spec/fixtures/factories.rb

```diff
@@ -13,6 +13,11 @@
     permissions %w(signin editor)
   end
 
+  factory :gds_editor, parent: :user do
```

Comment:
There's functionality in master for users to be able to log in, but not edit or publish documents. Since I was going MVP as fast as possible, I didn't pull across this functionality but it's in the factories. I've noted it on the board.